### PR TITLE
Add custom metrics section to javascript

### PR DESCRIPTION
**Pull request by ericlaz**
### What does this PR do?
Adds a section about how to add custom metrics to the JavaScript docs for the CI Visibility team.
### Motivation
There is no previous section about how to add custom metrics, many users have been asking about how to add custom metrics.

---

### Reviewer checklist
- [X] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/tests/javascript.md [hunk 1]
```diff
@@ -323,6 +350,28 @@ it('renders a hello world', () => {
...
+
+To add custom metrics to your tests, such as the memory allocations, use `cy.task('dd:addTags', { yourTags: 'here' })` in your test or hooks.
+
+For example:
+
+```javascript
+beforeEach(() => {
+  cy.task('dd:addTags', {
+    'before.each':
+    'certain.information'
```
Comment (juan-fernandez) [inline content/en/continuous_integration/tests/javascript.md:363]: I know this is like this in the custom tags section, but let's use the chance to change it to this: 
```suggestion
    'before.each': 'certain.information'
```

File: content/en/continuous_integration/tests/javascript.md [hunk 2]
```diff
@@ -323,6 +350,28 @@ it('renders a hello world', () => {
...
+  })
+})
+it('renders a hello world', () => {
+  cy.task('dd:addTags', {
+    'memory_allocation': 16,
+    'test.importance': 3
```
Comment (juan-fernandez) [inline content/en/continuous_integration/tests/javascript.md:369]: let's keep only `memory_allocation` here
```suggestion
    'memory_allocation': 16
```
and let's remove `test.importance` from the custom tags section

File: content/en/continuous_integration/tests/javascript.md [hunk 3]
```diff
@@ -154,6 +154,18 @@ You can add custom tags to your tests by using the current active span:
 To create filters or `group by` fields for these tags, you must first create facets. For more information about adding tags, see the [Adding Tags][1] section of the Node.js custom instrumentation documentation.
 
 [1]: /tracing/trace_collection/custom_instrumentation/nodejs?tab=locally#adding-tags
+### Adding custom metrics to tests
```
Comment (juan-fernandez) [inline content/en/continuous_integration/tests/javascript.md:157]:
```suggestion

### Adding custom metrics to tests
```

File: content/en/continuous_integration/tests/javascript.md [hunk 4]
```diff
@@ -212,6 +227,18 @@ You can add custom tags to your test by grabbing the current active span:
...
```
Comment (juan-fernandez) [inline content/en/continuous_integration/tests/javascript.md:230]:
```suggestion

### Adding custom metrics to tests
```

**Review by juan-fernandez — COMMENTED** [review]
looking good! Good job!

File: content/en/continuous_integration/tests/javascript.md [hunk 5]
```diff
@@ -323,6 +352,26 @@ it('renders a hello world', () => {
...
+To add custom metrics to your tests, such as the memory allocations, use `cy.task('dd:addTags', { yourTags: 'here' })` in your test or hooks.
+
+For example:
+
+```javascript
+beforeEach(() => {
+  cy.task('dd:addTags', {
+    'before.each': 'certain.information'
+  })
+})
```
Comment (juan-fernandez) [inline content/en/continuous_integration/tests/javascript.md:366]:
```suggestion
```

File: content/en/continuous_integration/tests/javascript.md [hunk 6]
```diff
@@ -323,6 +352,26 @@ it('renders a hello world', () => {
...
+it('renders a hello world', () => {
+  cy.task('dd:addTags', {
+    'memory_allocation': 16
```
Comment (juan-fernandez) [inline content/en/continuous_integration/tests/javascript.md:369]:
```suggestion
    'memory_allocations': 16
```

**Review by juan-fernandez — APPROVED** [review]

File: content/en/continuous_integration/tests/javascript.md [hunk 7]
```diff
@@ -323,6 +352,21 @@ it('renders a hello world', () => {
 
 To create filters or `group by` fields for these tags, you must first create facets. For more information about adding tags, see the [Adding Tags][105] section of the Node.js custom instrumentation documentation.
 
+### Adding custom metrics to tests
+
+To add custom metrics to your tests, such as the memory allocations, use `cy.task('dd:addTags', { yourTags: 'here' })` in your test or hooks.
```
Comment (urseberry) [inline content/en/continuous_integration/tests/javascript.md:357]:
```suggestion
To add custom metrics to your tests, such as memory allocations, use `cy.task('dd:addTags', { yourTags: 'here' })` in your test or hooks.
```

**Review by urseberry — COMMENTED** [review]
Thank you for submitting this! Just one minor nit.

**Review by urseberry — APPROVED** [review]

**Pull request merged**